crates/holochain/src/core/workflow/publish_dht_ops_workflow.rs

```diff
@@ -57,6 +65,92 @@ pub async fn publish_dht_ops_workflow(
     let mut success = Vec::with_capacity(to_publish.len());
     for (basis, list) in to_publish {
         let (op_hash_list, op_data_list): (Vec<_>, Vec<_>) = list.into_iter().unzip();
+
+        let mut have_enough_receipts = false;
+
+        // First, check to see if we can get the required validation receipts.
+        // TODO - we don't currently track the agents that generated the
+        //        receipts in our database, so we can't make use of the
+        //        exclude list, so we may get duplicate receipts.
+        if let Ok(bundle) = network
```

Comment:
`match` and log network errors other than no peers available?

Reply:
Done: https://github.com/holochain/holochain/pull/4796/commits/2a4269e557daf21cf8125f34451289766a206798